os/kernel/irq/irq_csection.c

```diff
@@ -245,6 +245,7 @@ irqstate_t enter_critical_section(void)
 			 * interrupt handler.
 			 */
 		} else {
+              int paused = false;
```

Comment:
The modification in this file alignment seems incorrect, could you please check?